### PR TITLE
Remove non important config from test environment

### PR DESCRIPTION
**Pull request by tomas-stefano**
Updating Rails makes the `1.hour` on the test environment to not
be there. Ideally we should fix this but we don't need this
specific config for our test env

### CODE REVIEW

**Review by brenetic — APPROVED** [review]
🚀

File: docker-compose.yml [hunk 1]
```diff
@@ -16,9 +16,9 @@ services:
...
-    links:
-      - runner-app-service-token-cache-app
-      - runner-app-datastore-app
+      #links:
+      #- runner-app-service-token-cache-app
```
Comment (njseeto) [inline docker-compose.yml:20]: we can delete these

**Review by njseeto — APPROVED** [review]
💯 🚀  just need to clean up a little bit but looks good otherwise

**Review by njseeto — APPROVED** [review]
🚀

**Pull request merged**